pm_yaml_config/pm_yaml_config.xml

```diff
@@ -0,0 +1,126 @@
+<tool id="pm_yaml_config" name="Creates yaml for pm_muairs" version="0.1.0" python_template_version="3.5">
+    <requirements>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        printf "poisson_r: $poisson_r \nname: $struct_name \ncharged: $charged \ngeom_steps: $geom_steps \nvdw_scale: $vdw_scale \ncalculator: $calculator_cond.calculator \ngeom_force_tol: $geom_force_tol \nclustering_method: $clustering_method \nclustering_hier_t: $clustering_hier_t \nout_folder: $out_folder \n">outputx.yaml &&
+        echo $calculator_cond.calculator &&
+        #if str($calculator_cond.calculator)=="uep":
+            printf "uep_gw_factor: $calculator_cond.uep_gw_factor \nuep_chden: $calculator_cond.uep_chden \n">>outputx.yaml &&
```

Comment:
Optional parameters like `uep_chden` should not be written if unset - results in an error:
```
Traceback (most recent call last):
  File "/home/qci88181/miniconda3/envs/__pymuonsuite@_uv_/lib/python3.9/site-packages/schema.py", line 405, in validate
    nvalue = Schema(svalue, error=e, ignore_extra_keys=i).validate(value, **kwargs)
  File "/home/qci88181/miniconda3/envs/__pymuonsuite@_uv_/lib/python3.9/site-packages/schema.py", line 463, in validate
    raise SchemaError(message, e.format(data) if e else None)
schema.SchemaError: validate_str(None) should evaluate to True

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/qci88181/miniconda3/envs/__pymuonsuite@_uv_/lib/python3.9/site-packages/pymuonsuite/schemas.py", line 112, in load_input_file
    params = param_schema.validate(params)
  File "/home/qci88181/miniconda3/envs/__pymuonsuite@_uv_/lib/python3.9/site-packages/schema.py", line 409, in validate
    raise SchemaError([message] + x.autos, [e.format(data) if e else None] + x.errors)
schema.SchemaError: Key 'uep_chden' error:
validate_str(None) should evaluate to True

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/qci88181/miniconda3/envs/__pymuonsuite@_uv_/bin/pm-muairss", line 10, in <module>
    sys.exit(main())
  File "/home/qci88181/miniconda3/envs/__pymuonsuite@_uv_/lib/python3.9/site-packages/pymuonsuite/muairss.py", line 336, in main
    params = load_input_file(args.parameter_file, MuAirssSchema)
  File "/home/qci88181/miniconda3/envs/__pymuonsuite@_uv_/lib/python3.9/site-packages/pymuonsuite/schemas.py", line 115, in load_input_file
    raise RuntimeError(message)
RuntimeError: Invalid input file params.yaml
Key 'uep_chden' error:
validate_str(None) should evaluate to True
```

Reply:
this took a long while to get right, but im pretty happy with the results